.github/workflows/branches.yml

```diff
@@ -47,3 +47,27 @@ jobs:
           IMAGES_POSTFIX: :master
         run: |
           make -C ${{ matrix.test-app }} release
+  test_vulcan:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: docker login
+        env:
+          DOCKER_TOKEN: ${{ secrets.DOCKER_TOKEN }}
+        run: docker login -p "${DOCKER_TOKEN}" -u etnaagent
+      - name: Run vulcan e2e test suite
+        id: app_first_test_run
+        env:
+          # Runs tests and builds images, but does not push (PUSH_IMAGES not set to 1)
+          IMAGES_PREFIX: etnaagent/
+          IS_CI: 1
+          CI_SECRET: ${{ secrets.CI_SECRET }}
+          IMAGES_POSTFIX: :master
+          RUN_E2E: 1
+        run: |
+          sudo rm -rf /usr/share/dotnet
+          sudo rm -rf /opt/ghc
+          sudo rm -rf /usr/local/lib/android
+          sudo rm -rf "/usr/local/share/boost"
+          sudo rm -rf "$AGENT_TOOLSDIRECTORY"
```

Comment:
per same code in master.yml and also production.yml workflows.  4/5 of these lines align with https://github.com/actions/runner-images/issues/2840#issuecomment-790492173 workaround for getting extra space in runner images.